src/stravalib/protocol.py

```diff
@@ -184,6 +192,7 @@ def exchange_code_for_token(
             expires_at (number of seconds since Epoch when the provided
             access token will expire)
         """
+        # The method returns: No rates present in response headers
```

Comment:
i'll hunt this down later but it relates to a discussion thread. it's just a message that's returned and isn't relevant to this PR. I'm leaving a note for later to be able to pinpoint where this is coming from.

Reply:
I think I read somewhere that Strava doesn't count authentication requests re. rate limits, so that may be the edge case that leads to us logging a warning. We can finetune the rate limiter to ignore authentication calls to prevent that warning from being logged.

Reply:
ok perfect. I'll turn this into an issue, and we can handle it separately!